Lib/test/test_builtin.py

```diff
@@ -1768,6 +1768,11 @@ def __getitem__(self, index):
         sum(([x] for x in range(10)), empty)
         self.assertEqual(empty, [])
 
+        xs = [complex(random.random() - .5, random.random() - .5)
+              for _ in range(10000)]
+        self.assertEqual(sum(xs), complex(sum(_.real for _ in xs),
```

Comment:
Maybe use `x` instead of `_`?

Reply:
Some people like the use `_` for unused variables, some don't.  Either way is okay.